app/scripts/modules/netflix/pipeline/stage/properties/create/propertyExecutionDetails.controller.js

```diff
@@ -17,7 +17,7 @@ module.exports = angular.module('spinnaker.netflix.pipeline.stage.property.detai
     let initialized = () => {
       $scope.detailsSection = $stateParams.details;
       $scope.properties = getPropertiesForAction($scope.stage.context);
-      $scope.originalProperties = extractProperties($scope.stage.context.originalProperties);
+      $scope.originalProperties = extractProperties($scope.stage.context.originalProperties || []);
```

Comment:
What do you think about modifying the `extractProperties` method to return the empty array instead?  I only ask, because the method right now implicitly returns `undefined` because of the way it's structured with the `if`-block.

I.e., maybe change to:

```
  let extractProperties = (propertyList) => {
    const result = [];
    if (propertyList && propertyList.length) {
      result.push(...propertyList.map(prop => prop.property));
    }

    return result;
  };
```

Or don't make a `const` and simply reassign `result`.